go.mod

```diff
@@ -25,9 +20,6 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	istio.io/api v0.0.0-20191015210738-bfa91e88abf1
-	k8s.io/api v0.0.0-20190817221950-ebce17126a01 // indirect
-	k8s.io/apimachinery v0.0.0-20190817221809-bf4de9df677c
-	k8s.io/client-go v10.0.0+incompatible
-	k8s.io/klog v0.4.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf // indirect
+	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
+	k8s.io/client-go v0.0.0-20191016111102-bec269661e48
```

Comment:
Do we need so newer dependency, whatever it should keep consistent with istio repo. Which is still 1.15

Reply:
See https://github.com/istio/istio/blob/156d828d379be772f8d04cac342210e2f04b97b2/go.mod#L168-L219. Looks like we need some extra care to keep k8s deps pinned to a specific version.

Reply:
IC

Reply:
But @howardjohn used `kubernetes-1.16.1` i think

Reply:
OK, Should be ok